agenta-backend/agenta_backend/services/db_manager.py

```diff
@@ -647,28 +715,34 @@ async def list_bases_for_app_id(
     """
 
     assert app_id is not None, "app_id cannot be None"
-    base_query = VariantBaseDB.find(VariantBaseDB.app.id == ObjectId(app_id))
-    if base_name:
-        base_query = base_query.find(VariantBaseDB.base_name == base_name)
-    bases_db = await base_query.sort("base_name").to_list()
-    return bases_db
+    async with db_engine.get_session() as session:
+        query = select(VariantBaseDB).filter_by(app_id=uuid.UUID(app_id))
+        if base_name:
+            query = query.filter_by(base_name=base_name)
+
+        result = await session.execute(query.order_by(VariantBaseDB.base_name.asc()))
+        bases = result.scalars().all()
+        return bases
 
 
-async def list_variants_for_base(base: VariantBaseDB) -> List[AppVariantDB]:
+async def list_variants_for_base(base: VariantBaseDB):
```

Comment:
why removing the the return type ? 
if needed let's adjust the docstrings everywhere too